libraries/standard/mqtt/include/mqtt.h

```diff
@@ -246,7 +257,12 @@ MQTTStatus_t MQTT_Unsubscribe( MQTTContext_t * const pContext,
 MQTTStatus_t MQTT_Disconnect( MQTTContext_t * const pContext );
 
 /**
- * @brief Loop to receive packets from the transport interface.
+ * @brief Loop to receive packets from the transport interface. Handles keep
+ * alive.
+ *
+ * @note This function assumes that pContext->callbacks.getTime is defined.
```

Comment:
(Minor) Suggest re-wording the text to mention that `getTime` is a requirement for this function. So, suggest something like:
"_This function requires a valid definition of pContext->callbacks.getTime function. It is required for supporting the keep-alive mechanism that sends ping requests at configured keep-alive intervals of idle connection time._"

Reply:
related to PR #990